packages/bento-design-system/src/defaultTheme.ts

```diff
@@ -0,0 +1 @@
+import "./defaultTheme.css";
```

Comment:
didn't find a better way to generate a `defaultTheme.css`.
if we include `defaultTheme.css.ts` directly as an entry in the tsup config, we get a `defaultTheme.css.css`